e2e/plugin-e2e/various-suite/mocks/mysql.mocks.ts

```diff
@@ -0,0 +1,72 @@
+export const normalTableName = 'normalTable';
```

Comment:
Can we change the name of the folder `various-suite` to `mysql`? Will make it easier to find tests, but it will also allow different plugins to create their own Playwright projects with custom config, feature toggles etc.

Reply:
done